### ISSUE TITLE
How to implement?

### ISSUE THREAD

**Issue by djmtype** [11ty/eleventy-img#1]
@zachleat Once we add the plugin and configuration, what syntax do we need to add to our template / markdown file to get the responsive image size markup and files?

**Comment by zachleat** [comment]
I added a couple of examples to the README https://github.com/11ty/eleventy-img

Hope that helps!

**Comment by indcoder** [comment]
@zachleat Can one of the example include markdown please? While the blog collections page is in Nunjucks, its the individual blog posts that have a lot of images and LightHouse keeps complaining about the first load stats on the page. I'm using markdown-it for the .md files

**Comment by ajenkins** [comment]
@indcoder I was wondering how to use this plugin with Markdown files too. Not sure if this is the best solution, but here's how I did it.

The first thing to do is use Nunjucks as your template engine for Markdown files. This will allow you to use Nunjucks shortcodes directly in your Markdown files. Personally, I was worried that this would break things, but it seems to "just work." You can see [how to do that here](https://www.11ty.dev/docs/config/#default-template-engine-for-markdown-files). You'll need to add a new property to the return object in your .eleventy.js file.

Then you can add Nunjucks shortcodes directly into your Markdown files. It seems to mostly just work, but make sure your Nunjucks code is separated by a full newline on either side, because anything immediately above or below it will get interpreted as HTML instead of Markdown. Here's an example:
```markdown
This will be interpreted as normal Markdown.

<p>
  {% myImage "./src/images/cat.jpg", "photo of my cat" %}
</p>

This will _also_ be interpreted as normal Markdown.
```

**Comment by indcoder** [comment]
Yes, I have just verified it. But I also went a different direction because image optimization on the web is a multi pronged approach. 

We have based our blog on one that was doing it holistically.
https://github.com/HackerspaceMumbai/eleventy-high-performance-blog

Thank you for putting this solution out. Infact I think should be part of the examples....because a majority of a blog's content will be in markdown. cc @zachleat
